client/activity_reports/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * This file is part of Superdesk.
+ *
+ * Copyright 2016 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code, or
+ * at https://www.sourcefabric.org/superdesk/license
+ */
+
+import * as svc from './services';
+import * as directive from './directives';
+
+
+function cacheIncludedTemplates($templateCache) {
+    $templateCache.put('activity-report-parameters.html', require('./views/activity-report-parameters.html'));
```

Comment:
There is grunt task `ngtemplates` which can automatically convert all html files and put them in `$templateCache`. It is implemented in superdesk-client-core. It is cool feature, you don't need to worry about templates.